proposals/description/tip_118.ts

```diff
@@ -0,0 +1,497 @@
+      description: 'Revoke PCV_CONTROLLER_ROLE from ethPSMFeiSkimmer'
+    },
+
+    //// Activate DAI PSM skimmer and burn Fei
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('PCV_CONTROLLER_ROLE'), addresses.daiFixedPricePSMFeiSkimmer],
+      description: 'Grant PCV_CONTROLLER_ROLE to daiFixedPricePSMFeiSkimmer'
+    },
+    {
+      target: 'daiFixedPricePSMFeiSkimmer',
+      values: '0',
+      method: 'skim()',
+      arguments: (addresses) => [],
+      description: 'Burn excess Fei on the DAI PSM. Will burn ~95M FEI'
+    },
+
+    ///// Send VOLT to it's holding ERC20 deposit
+    {
+      target: 'volt',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: (addresses) => [addresses.voltHoldingPCVDeposit, '10000000000000000000000000'],
+      description: 'Move 10M VOLT from the DAO timelock to the VOLT holding PCV deposit'
+    },
+
+    ///// Send gOHM to it's holding ERC20 deposit
+    {
+      target: 'gOHM',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: (addresses) => [addresses.gOHMHoldingPCVDeposit, '577180000000000000000'],
+      description: 'Send 577 gOHM from the DAO timelock to the gOHM holding PCV deposit'
+    },
+
+    // Withdraw excess TRIBE from reward system
+    {
+      target: 'erc20Dripper',
+      values: '0',
+      method: 'withdrawERC20(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribe, addresses.core, '2508838570600494412126519'],
+      description: 'Withdraw 2.5M TRIBE from the ERC20 Dripper to the Core Treasury'
+    },
+    {
+      target: 'votiumBriber3Crvpool',
+      values: '0',
+      method: 'withdrawERC20(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribe, addresses.core, '1725076846586787179639648'],
+      description: `
+      Withdraw all TRIBE from 3CRV Votium briber contract to the Core Treasury. 
+      
+      Withdraw is made up of 200k TRIBE pre-existing on this contract and then an additional 1.5M TRIBE 
+      that was harvested from the stakingTokenWrapperBribe3Crvpool.
+      `
+    },
+    {
+      target: 'votiumBriberD3pool',
+      values: '0',
+      method: 'withdrawERC20(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribe, addresses.core, '232096077769383622085234'],
+      description: `
+      Withdraw all TRIBE from D3 Votium briber contract to the Core Treasury. 
+      
+      Withdraw is made up of 230k TRIBE that was harvested from the stakingTokenWrapperBribeD3pool.
+      `
+    },
+    {
+      target: 'rewardsDistributorAdmin',
+      values: '0',
+      method: '_grantComp(address,uint256)',
+      arguments: (addresses) => [addresses.core, '150000000000000000000000'],
+      description: `Withdraw excess 150k TRIBE from Rari delegator contract. There is 164k excess, leaving behind a buffer of 14k`
```

Comment:
why do we leave a buffer of 14k if we know the excess is 164k ?

Reply:
Being conservative. This was calculated off-chain by looking at the speeds of the auto reward distributor over time and inferring from that. We're reasonably confident, but want to leave a buffer in case.

Downside of leaving behind some additional TRIBE is less than any potential downside of some users not being able to claim imo